osa/paths.py

```diff
@@ -232,10 +232,14 @@ def destination_dir(concept: str, create_dir: bool = True) -> Path:
     nightdir = utils.date_to_dir(options.date)
 
     if concept == "MUON":
-        directory = Path(cfg.get(options.tel_id, concept + "_DIR")) / nightdir / options.prod_id
-    elif concept in {"DL1AB", "DATACHECK"}:
+        directory = Path(cfg.get(options.tel_id, "DL1_DIR")) / nightdir / options.prod_id / "muons"
+    elif concept == "INTERLEAVED":
+        directory = Path(cfg.get(options.tel_id, "DL1_DIR")) / nightdir / options.prod_id / "interleaved"
+    elif concept == "DATACHECK":
+        directory = Path(cfg.get(options.tel_id, "DL1_DIR")) / nightdir / options.prod_id / "datacheck"
```

Comment:
datacheck files should be placed inside the `dl1b_prod_id` (for us `tailcut84`) since they are based on dl1b files